diesel/src/pg/types/array.rs

```diff
@@ -48,7 +48,10 @@ where
                 } else {
                     let (elem_bytes, new_bytes) = bytes.split_at(elem_size as usize);
                     bytes = new_bytes;
-                    T::from_sql(Some(elem_bytes))
+                    T::from_sql(Some(&PgValue {
+                        data: elem_bytes.to_vec(),
+                        oid: 1, //TODO FIXFIXFIX
```

Comment:
Oh this is interesting. We could probably just stick a reference to the metadata lookup on `PgValue` but I'm not sure I want to do that.

Reply:
Actually no, we don't need to do any of that. `_oid` above is the right value.

Reply:
Fixed